src/main/java/com/minecolonies/coremod/colony/requestsystem/management/handlers/UpdateHandler.java

```diff
@@ -36,4 +36,9 @@ public static void handleUpdate(@NotNull final IStandardRequestManager manager)
               manager.setCurrentVersion(s.updatesToVersion());
           });
     }
+
+    public static int getCurrentVersion()
+    {
+        return steps.stream().max(Comparator.comparing(IUpdateStep::updatesToVersion)).get().updatesToVersion();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "Optional#isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3655)